SPECS/mft_kernel/mft_kernel.spec

```diff
@@ -228,6 +228,9 @@ find %{buildroot} -type f -name \*.ko -exec %{__strip} -p --strip-debug --discar
 %endif
 
 %changelog
+* Wed Feb 19 2025 Chris Co <chrco@microsoft.com>
```

Comment:
Missing version & release. Need to find out why rpmdev-bumpspec didn't automatically add it...